README.md

```diff
@@ -168,7 +169,7 @@ If you like this project please share with your partners.
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank @Microverse online school.
+I would like to thank @Microverse online school and Behance for the Design.
```

Comment:
- [ ] Kidly consider giving appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).